### PR TITLE
a better list of teas

### PR DESCRIPTION
**Pull request by easement**
I parsed a tea catalog recently for a project and noticed it had a wider selection of items and it could be useful for this project.

### CODE REVIEW

File: data/foods/tea.json [hunk 1]
```diff
@@ -1,182 +1,369 @@
...
+     "Kyo Bancha Organic",
+     "Lapsang Souchong",
+     "Lemon Grass, Coarse Cut",
+     "Lemon Verbena",
+     "Lover's Leap Estate Pekoe 1",
+     "Lukwah Estate TGFBOP Spl.",
+     "Lumbini Estate FBOPF Ex. Sp.",
+     "Lumbini Estate FBOPF Ex. Spl.",
+     "Lung Jing Te Ji",
+     "Lung-Ching:",
```
Comment (hugovk) [inline data/foods/tea.json]: Remove trailing  `:`

File: data/foods/tea.json [hunk 2]
```diff
@@ -1,182 +1,369 @@
...
+     "Superior Grade",
+     "Sweet Almond Green Tea",
+     "Sweet Honey Black Tea",
+     "Sweet Orange",
+     "Tai Ping Hou Kui",
+     "Tea Bank Estate FBOPF Ex. Sp.",
+     "Tea Bank FBOPF Ex. Sp.",
+     "Thurbo Estate FTGFOP1 Moonlight",
+     "Tie-Guan-Yin",
+     "Tie-Guan-Yin:",
```
Comment (hugovk) [inline data/foods/tea.json]: Remove this line. It duplicates the previous line and adds a `:`.

File: data/foods/tea.json [hunk 3]
```diff
@@ -1,182 +1,369 @@
...
+     "Richmond Park Blend",
+     "River Shannon Breakfast Blend",
+     "Roasted Oolong",
+     "Roasted Yerba Maté",
+     "Rooibos Apple Cider",
+     "Rooibos Candied Almond",
+     "Rooibos Vanilla",
+     "Rose Congou",
+     "Rose Hips",
+     "Rote Grütze (Red Groats)",
```
Comment (hugovk) [inline data/foods/tea.json]: The following line is "Rote Grütze", so maybe change this to "Red Groats"?

**Comment by easement** [comment]
@hugovk

1. added back in the original set
2. title cased them.
3. alphabetized them.
4. took out the redundant `Tea` in the names
5. cleaned out the `:`s

File: data/foods/tea.json [hunk 4]
```diff
@@ -1,27 +1,46 @@
 {
-  "description": "types of tea",
+  "description": "types of",
```
Comment (hugovk) [inline data/foods/tea.json]: Maybe this one line can keep the " tea" suffix :)

Comment (easement) [inline data/foods/tea.json]: sloppy find/replace on my part. good catch.

Comment (easement) [inline data/foods/tea.json]: done.

**Comment by easement** [comment]
it looked like it was getting almost there, so I fixed the description and did a rebase to keep the history clean.

**Comment by easement** [comment]
Anything else needed?

**Pull request merged**